common/xrdp_client_info.h

```diff
@@ -33,6 +33,17 @@ struct monitor_info
     int is_primary;
 };
 
+/* xrdp keyboard overrids */
+struct xrdp_keyboard_overrides
+{
+    int type;
+    int subtype;
+    int layout;
+    int original_type;
+    int original_subtype;
+    int original_layout;
```

Comment:
I don't think the `original_*` values are needed(?)

Reply:
I had added it in case the original client information was needed after the override.
This is not needed for this PR and will be removed.